src/htmlminifier.js

```diff
@@ -796,6 +796,21 @@
       buffer.length = Math.max(0, index);
     }
 
+    function squashTrailingWhitespace(nextTag) {
```

Comment:
Could you please add a comment as to what "squash" means here (since it's kind of ambiguous).